src/commcare_cloud/ansible/roles/formplayer/tasks/rollback.yml

```diff
@@ -34,3 +34,10 @@
       state: link
       force: yes
       follow: no
+
+  - name: Re-read supervisor config
+    command: supervisorctl reread
+
+  - name: Restart formplayer service
+    supervisorctl: name=all state=restarted
+    # on a monolith this actually restarts all services
```

Comment:
The commands used to restart here are a simplified version of what was done previously
https://github.com/dimagi/commcare-cloud/blob/b28a81770b673a7cca4298a44b25f85592740b06/src/commcare_cloud/fab/operations/supervisor.py#L104-L118

That code was originally added in https://github.com/dimagi/commcare-cloud/commit/010b20d80852c6ddf4c4dd3ad09037c80cbb56c1, which has no explanation of why it used so many commands to perform the restart.